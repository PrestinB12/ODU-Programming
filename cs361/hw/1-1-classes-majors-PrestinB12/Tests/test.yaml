test:
  timelimit: 10
  launch: ./listMajor
  stderr: false
dueDate: 09/07/2023
latePenalties: [10, 20, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["readingList.h", "readingList.cpp"]
  instructorFiles:
    exclude: ["readingList.h", "readingList.cpp"]
