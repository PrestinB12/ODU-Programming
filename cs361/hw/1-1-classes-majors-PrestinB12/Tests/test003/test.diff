10,11c10
<    anth201 (geo101,geo101,math101)
< **Memory leaks detected: 70
---
>    anth201 (bio101,geo101,math101)
