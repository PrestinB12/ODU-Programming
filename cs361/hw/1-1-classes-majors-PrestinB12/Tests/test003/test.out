Semester 1
   cs101 ()
   fren101 ()
   hist499 ()
Semester 2
   bio101 (fren101)
   geo101 (hist499)
   math101 (cs101)
Semester 3
   anth201 (geo101,geo101,math101)
**Memory leaks detected: 70
